File: Assignment37_1.c
// Write a program to accept a string and accept on character and check whether that character is present in string or not

#include<stdio.h>
#include<stdbool.h>

bool Search(char *str, char ch)
{
    bool bFlag = false;

    while(*str != '\0')
    {
        if(*str == ch)
        {
            bFlag = true;
            break;
        }
        str++;
    }
    return bFlag;
}

int main()
{
    char Arr[] = {'\0'};
    bool bRet = false;
    char cValue = '\0';

    printf("Enter a String: \n");
    scanf("%[^'\n']s",Arr);

    printf("Enter a character: \n");
    scanf(" %c",&cValue);

    bRet = Search(Arr, cValue);
    
    if(bRet == true)
    {
        printf("%c is present\n",cValue);
    }
    else
    {
        printf("%c is not present\n",cValue);
    }

    return 0;
}

File: Assignment37_1.java
// Write a program to accept a string and accept on character and check whether that character is present in string or not
import java.util.*;

class Logic
{
    public boolean CheckCharacter(String str, char ch)
    {
        char Arr[] = str.toCharArray();
        int i = 0;
        boolean bFlag = false;

        for(i = 0; i < Arr.length; i++)
        {
            if(Arr[i] == ch)
            {
                bFlag = true;
            }            
        }
        return bFlag;
    }
}


class Assignment37_1
{
    public static void main(String A[])
    {
        Scanner scanobj = new Scanner(System.in);

        System.out.println("Enter a string: ");
        String sobj = scanobj.nextLine();

        System.out.println("Enter a character: ");
        char cValue = scanobj.next().charAt(0);

        Logic lobj = new Logic();
        boolean bRet = lobj.CheckCharacter(sobj, cValue);
        
        if(bRet == true)
        {
            System.out.println(cValue+" is present");
        }
        else
        {
            System.out.println(cValue+" is not present");
        }
    }
}